domain/schema/controller/sql/0026-secret-backend-immutable-trigger.PATCH.sql

```diff
@@ -0,0 +1,25 @@
+-- When merging into main, update the file domain/schema/controller.go:
+--  - triggersForImmutableTable "secret_backend" needs to be updated to below condition and message
+--  - this file needs to be removed
+
+DROP TRIGGER IF EXISTS trg_secret_backend_immutable_update;
```

Comment:
```suggestion
DROP TRIGGER trg_secret_backend_immutable_update;
```

We know it exists, don't suggest otherwise.